features/autorisatie/bewoning/raadpleeg-bewoning-met-periode/autorisatie.feature

```diff
@@ -62,7 +62,7 @@ Functionaliteit: autorisatie voor het gebruik van de API BewoningMetPeriode
       | code     | unauthorized                                                                           |
       | instance | /haalcentraal/api/bewoning/bewoningen                                                  |
 
-    Scenario: Gemeente raadpleegt bewoning van een adresseerbaar object binnen de gemeente en de bewoner is nu niet meer ingeschreven in de gemeente
+    Scenario: Gemeente raadpleegt bewoning van een adresseerbaar object binnen de gemeente waarvan de bewoner nu niet meer is ingeschreven in de gemeente
```

Comment:
Deze scenario doet niet meer toe. Er wordt alleen gekeken of het adresseerbaar object ooit in de gemeente van de afnemer heeft gelegen